src/components/common/Navbar/Navbar.tsx

```diff
@@ -15,6 +14,8 @@ import type { AnchorHTMLAttributes } from 'react'
 import type { SearchInputRef } from '@faststore/ui'
 import type { StoreCollectionQuery } from '@generated/graphql'
 
+import PostalCodeBar from '../PostalCode/PostalCodeBar'
```

Comment:
Do you mind moving this import to line 5, where the other was removed?

Reply:
No problem, I'll change it.

Reply:
I re-opened this issue because it still hasn't been moved yet, perhaps you forgot?